core/src/idx/trees/mtree.rs

```diff
@@ -1,6 +1,8 @@
 use std::cmp::Ordering;
 use std::collections::btree_map::Entry;
-use std::collections::{BTreeMap, BinaryHeap, HashMap, HashSet, VecDeque};
+#[cfg(debug_assertions)]
+use std::collections::HashMap;
```

Comment:
Just curious about what you think about `hashbrown::HashMap`. I've noticed it to be much faster than the default hashmap when benchmarking for skv

Reply:
If hashdos resistant is not something we need you could look at trying a different hashing algorithm. The default SipHash that rust uses is pretty slow.

Reply:
Confirmed to be the default implementation since Rust 1.36
https://crates.io/crates/hashbrown